janusgraph-core/src/main/java/org/janusgraph/graphdb/tinkerpop/JanusGraphFeatures.java

```diff
@@ -78,7 +78,7 @@ public static JanusGraphFeatures getFeatures(StandardJanusGraph graph, StoreFeat
 
         @Override
         public boolean supportsMapValues() {
-            return false;
+            return true;
```

Comment:
Yes, for example in `org.apache.tinkerpop.gremlin.process.computer.GraphComputerTest` the shouldSucceedWithProperTraverserRequirements and shouldFailWithImproperTraverserRequirements tests throw IllegalArgumentExceptions (`Property value [{...}] is of type class java.util.HashMap is not supported`) with supportsMapValues=false and without the associated serialization implementation. See [attached stack trace](https://github.com/JanusGraph/janusgraph/files/762082) for details.